### ISSUE TITLE
Public account script

### ISSUE THREAD

**Issue by raitoakpro** [MinoLiu/Let-s-panda#16]
I used to use your script for public account on tampermonkey 2 years ago, it worked well on all browser I was using (opera, ff, chrome, edge). However, till yesterday it suddenly got blank page. Are there any errors with the script now?

**Comment by MinoLiu** [comment]
Hi, I just tried to log in again with FF and chrome, and both work well. 
You could try to clear the cache and cookies at exhentai.org, e-hentai.org, forums.e-hentai.org.
Then log in using this script again.

**Comment by raitoakpro** [comment]
Thx for reply, I tried to clear all of them but still got blank page. Just for sure, I use script for public account, so is it necessary to login ? i use this one ExHentai.org自动登录... . I also try  ExHentai / E-Hentai Auto Login, but it got same issue as well.
![image](https://user-images.githubusercontent.com/91532132/194740776-6b5033d9-c3e8-40c2-821b-36510c055d16.png)

**Comment by MinoLiu** [comment]
@raitoakpro Sorry, this script only provides an interface to let you log in.
![image](https://user-images.githubusercontent.com/17308188/194741786-6176b4ff-c65b-46e6-9af2-797859619e2d.png)
So if you have an account you can just follow the instructions to log in.

It looks like your public account script is no longer working.

**Comment by raitoakpro** [comment]
@MinoLiu Seems like that, of course I have tried your Let-s-panda as well. However, I watched your comments in other issue posts and u said that account created from Asia isn't probably accepted. btw my account has created for 3 years and still couldn't access exh. I also tried all ways people said on github and reddit but it seems impossible for me, so I decided to use public account instead. And now public account doesn't work too... How sad! thx you anyway.
